packages/suite/src/components/wallet/TransactionItem/TransactionItem.tsx

```diff
@@ -105,29 +105,34 @@ export const TransactionItem = memo(
         );
 
         const { type, targets, tokens, internalTransfers } = transaction;
+
         const isUnknown = type === 'unknown';
+        // Filter out internal transfers that are instant staking transactions
```

Comment:
how about unstaking 
![Screenshot 2024-07-12 at 12 36 11](https://github.com/user-attachments/assets/47282b98-d24e-4ce5-a9fe-8c916695c2a4)

Reply:
Instant unstaking transfers have _from: PoolAddress_ and _to: accountAddress_. So, we've already had these transfers and shown them. From my understanding, we don't need any additional filtering for instant unstaking.

Reply:
Ah, so you do not need to claim instant unstake, right?

Reply:
yes, these blockbook changes only affect staking transactions.